pages/your_grants/view_applicants/manage.tsx

```diff
@@ -302,7 +302,7 @@ function ManageGrant() {
         pb={8}
         px={10}
       >
-        <Breadcrumbs path={path} />
+        <Breadcrumbs path={path} id={applicationData?.grant?.id} />
```

Comment:
It seems on `manage` page the `View Application` link is not working properly. Not getting the `applicationId` query param. It's null.